CONTRIBUTING.md

```diff
@@ -92,6 +92,15 @@ Maintainers are:
  * @stu-gott
  * @vladikr
 
+### Becoming a memeber
+
+Contributors that frequently contribute to the project may ask to join the
+kubevirt organization.
+Please send a request to join by sending an email to our
```

Comment:
the request should include a self-introduction, and an explanation why you should be a member.

Reply:
Done